Assets/Scripts/API/FactionFile.cs

```diff
@@ -593,6 +593,12 @@ public enum GuildGroups
             Witches = 22,
             Vampires = 23,
             Orsinium = 24,
+            GGroup25 = 25,
```

Comment:
Btw this is not really needed. Users can easily write `const GuildsGroups MyRenegadeGuild = (GuildGroups)25;` themselves, no need to extend the enum here. No need to revert it either, but also let's not keep adding every few versions.